aliasing/api/character.py

```diff
@@ -432,6 +458,29 @@ def reset(self):
         """
         return self._cc.reset()
 
+    def full_str(self, include_name: bool = False):
+        """
+        Returns a string representing the full custom counter.
+
+        :param bool include_name: If the name of the counter should be included. Defaults to False.
+        :returns: A string representing all components of the counter.
+        :rtype: str
+
+        Example:
+
+        >>> full_str('Bardic Inspiration')
+        ◉◉◉◉
+        Resets On: Long Rest
+        >>> full_str('Bardic Inspiration', True)
+        Bardic Inspiration
+        ◉◉◉◉
+        Resets On: Long Rest
```

Comment:
```suggestion
        "◉◉◉◉\n"
        "**Resets On**: Long Rest"
        >>> full_str('Bardic Inspiration', True)
        "**Bardic Inspiration**\n"
        "◉◉◉◉\n"
        "**Resets On**: Long Rest"
```
doctests should show the `repr` of evaluating each line (unless it's a print)